flake.d/README.md

```diff
@@ -0,0 +1,74 @@
+This directory contains support files for `flake.nix`, which is packaging to allow LMQL and its dependencies to be installed anywhere [Nix](https://nixos.com/) is available, even if Python or Node.js wasn't previously available at all!
+
+This includes installation of a copy of llama.cpp.
```

Comment:
Can this be made optional?

Reply:
It can, if you think it's worth it. Would mean a proliferation of entrypoints -- one devShell _with_ llama.cpp, another without, etc.

Reply:
This is done. Our `nix run` endpoints are now (as legacy packages):

```none
    │   ├───default omitted (use '--all-systems' to show)
    │   ├───llamaDotCpp omitted (use '--all-systems' to show)
    │   ├───lmql-docs omitted (use '--all-systems' to show)
    │   ├───playground omitted (use '--all-systems' to show)
    │   ├───playground-all omitted (use '--all-systems' to show)
    │   ├───playground-basic omitted (use '--all-systems' to show)
    │   ├───playground-hf omitted (use '--all-systems' to show)
    │   ├───playground-llamaCpp omitted (use '--all-systems' to show)
    │   ├───python-all omitted (use '--all-systems' to show)
    │   ├───python-basic omitted (use '--all-systems' to show)
    │   ├───python-hf omitted (use '--all-systems' to show)
    │   └───python-llamaCpp omitted (use '--all-systems' to show)
```

...and as apps:

```none
│   │   ├───lmtp-server: app
│   │   ├───lmtp-server-all: app
│   │   ├───lmtp-server-basic: app
│   │   ├───lmtp-server-hf: app
│   │   └───lmtp-server-llamaCpp: app
```

...and our `nix develop` endpoints are:

```none
│       ├───default: development environment 'interactive-python3-3.10.12-environment'
│       ├───lmql-all: development environment 'interactive-python3-3.10.12-environment'
│       ├───lmql-basic: development environment 'interactive-python3-3.10.12-environment'
│       ├───lmql-hf: development environment 'interactive-python3-3.10.12-environment'
│       ├───lmql-llamaCpp: development environment 'interactive-python3-3.10.12-environment'
│       └───minimal: development environment 'minimal-dev-shell'
```